### PR TITLE
feat: add `News` page

### PR DESCRIPTION
**Pull request by brenopolanski**
## Summary

**Screenshot 1:**

Screenshot from 2020-07-09 18-16-29

**Screenshot 2:**

Screenshot from 2020-07-09 18-16-32

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: src/domains/news/pages/News/News.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	const [currentPage, setCurrentPage] = useState(1);
+	const crumbs = [
+		{
+			route: "portfolio",
+			label: "Go back to Portfolio",
+		},
+	];
+
+	return (
+		<div data-testid="News" className="flex flex-col min-h-screen -m-5 bg-theme-neutral-100">
```
Comment (dated) [inline src/domains/news/pages/News/News.tsx:21]:
```suggestion
		<div data-testid="News" className="flex flex-col min-h-screen -m-5 bg-theme-neutral-contrast">
```

Same color, but aliased.

Comment (brenopolanski) [reply src/domains/news/pages/News/News.tsx]: done @dated

**Comment by faustbrian** [comment]
@brenopolanski conflicts

File: src/domains/news/components/NewsOptions/NewsOptions.tsx [hunk 2]
```diff
@@ -16,10 +17,14 @@ export const NewsOptions = ({ categories, selectedAssets }: Props) => {
...
-		<div className="p-8 border-2 rounded-lg border-theme-primary-contrast" data-testid="NewsOptions">
+		<div
+			className="p-8 border-2 rounded-lg bg-theme-background border-theme-primary-contrast"
+			data-testid="NewsOptions"
+		>
 			<div className="flex flex-col space-y-10">
-				<div className="flex justify-end">
-					<HeaderSearchBar />
+				<div className="flex items-center px-4 py-6 rounded shadow-xl">
+					<Input className="border-none shadow-none" placeholder="Search" />
```
Comment (goga-m) [inline src/domains/news/components/NewsOptions/NewsOptions.tsx:26]: Hey @brenopolanski I think if you add a `padding: 1rem 0.75rem` as a styled css (py-4 and px-3 will not override the existing styling) will fix the glitching issue @faustbrian mentioned.

Comment (brenopolanski) [reply src/domains/news/components/NewsOptions/NewsOptions.tsx]: Thanks @goga-m i solved using a styled wrapper :+1:

**Pull request merged**